### PR TITLE
Add pt-BR top 4 editor keys

### PR DESCRIPTION
**Pull request by alexheretic**
@johnwss can you help translate the rest of the new editor pairs?

### CODE REVIEW

File: translated-pairs/en-replace.pt-BR.pairs [hunk 1]
```diff
@@ -296,16 +296,16 @@ Número total de frases de código avaliadas
...
 Copy
-Copy
+Copiar
 
 Paste
-Paste
+Colar
 
 Cut line/selection
 Cut line/selection
```
Comment (johnwss) [inline translated-pairs/en-replace.pt-BR.pairs]: Cortar linha/seleção

File: translated-pairs/en-replace.pt-BR.pairs [hunk 2]
```diff
@@ -345,4 +345,3 @@ Autocomplete apply
 
 Autocomplete close
 Autocomplete close
```
Comment (johnwss) [inline translated-pairs/en-replace.pt-BR.pairs]: Fechar autocompletar

**Review by johnwss — CHANGES_REQUESTED** [review]
Simple sentences! ( :

**Comment by alexheretic** [comment]
Thanks for the suggestions. Can you raise a pr for the others?

**Pull request merged**